include/platform/radio.h

```diff
@@ -207,7 +262,8 @@ void otPlatRadioSetShortAddress(otInstance *aInstance, uint16_t aShortAddress);
  * @retval ::kThreadError_None     Successfully enabled.
  * @retval ::kThreadError_Failure  The radio could not be enabled.
  */
-ThreadError otPlatRadioEnable(otInstance *aInstance);
+ThreadError otPlatRadioEnable(otInstance *aInstance, otPlatRadioReceiveDone receiveCallback,
+                              otPlatRadioTransmitDone transmitCallback);
```

Comment:
My options were either pass it in here, make a new function to set these, or set each individual callback in their corresponding receive/transmit functions. I chose this because it seemed the smallest change.